packages/resource-deployment/scripts/function-app-create.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+set -euo pipefail
+
+exitWithUsageInfo() {
+    echo "
+Usage: $0 -s <subscription name or id> -r <resource group> -l <location>
+"
+    exit 1
+}
+
+export subscription=""
+export resourceGroupName=""
+export location=""
+export functionAppName
+export appInsightsName
+export storageAccountName
+
+# Set default ARM Function App template files
+templateFilePath="../templates/function-app-template.json"
+
+# Read script arguments
+while getopts "s:r:l:"  option; do
+    case $option in
+		s) subscription=${OPTARG} ;;
+		r) resourceGroupName=${OPTARG} ;;
+		l) location=${OPTARG} ;;
+		*) exitWithUsageInfo ;;
+		esac
+done
+
+if [ -z "$subscription" ] || [ -z "$resourceGroupName" ] || [ -z "$location" ]; then
```

Comment:
expect appInsightsName to be set as well

Reply:
We can't do that, the only way to do it is the same as we do app-insights-create.sh, we set it in the template file.
so we're gonna use the same name as in app-insights-create.template.json and we are **not** going to create a new app insights resource